pkg/alertmanager/amcfg.go

```diff
@@ -2017,6 +2057,19 @@ func (tc *webexConfig) sanitize(amVersion semver.Version, logger log.Logger) err
 	return tc.HTTPConfig.sanitize(amVersion, logger)
 }
 
+func (tc *msteamsConfig) sanitize(amVersion semver.Version, logger log.Logger) error {
+	msteamsAllowed := amVersion.GTE(semver.MustParse("0.25.1"))
```

Comment:
nit!
since you're not using the `msteamsAllowed` variable in any other place, IMO you can avoid the variable allocation and use the amversion.GTE straight away on the if statement.

Reply:
updated.